lib/spaceship/tunes/tunes_client.rb

```diff
@@ -393,6 +393,36 @@ def price_tier(app_id)
       end
     end
 
+    # Returns an array of all available pricing tiers
+    # @return [Array] the PricingTier objects
+    # [{
+    #   "tierStem": "0",
+    #   "tierName": "Free",
+    #   "pricingInfo": [{
+    #       "country": "United States",
+    #       "countryCode": "US",
+    #       "currencySymbol": "$",
+    #       "currencyCode": "USD",
+    #       "wholesalePrice": 0.0,
+    #       "retailPrice": 0.0,
+    #       "fRetailPrice": "$0.00",
+    #       "fWholesalePrice": "$0.00"
+    #     }, {
+    #     ...
+    # }, {
+    # ...
+    def pricing_tiers
```

Comment:
I got your point right after writing the comment above and tried the same thing you suggested but ran into the same issue.

It looks like there was simply no need for unauthorized ITC requests so far. I'll state this in the pricing_tiers method documentation for now.

To be honest, I don't know how many requests ITC provides that don't require authorization, but it would be cool to be able to do this in future without workarounds (thanks for that). This means re-thinking the base.rb, tunes_client.rb etc. to provide an interface not requiring the client implementation to be implemented by subclasses, which is not trivial for these "rare" cases. Maybe I can come up with a proper solution if I find time, but that's a different story.

Anyway, thank you very much for your help with this @lacostej!